crates/ironrdp-session/src/x224/mod.rs

```diff
@@ -60,32 +62,33 @@ impl Processor {
         }
     }
 
-    pub fn get_svc<T: StaticVirtualChannel + 'static>(&mut self) -> Option<&T> {
+    pub fn get_svc_processor_downcast_ref<T: StaticVirtualChannelProcessor + 'static>(&mut self) -> Option<&T> {
```

Comment:
Nit: I think it’s reasonable to make it less verbose in this place
```suggestion
    pub fn get_svc_processor<T: StaticVirtualChannelProcessor + 'static>(&mut self) -> Option<&T> {
```